.test-infra/jenkins/job_PostCommit_Java_Jpms_Java11.groovy

```diff
@@ -39,7 +39,7 @@ PostcommitJobBuilder.postCommitJob('beam_PostCommit_Java_Jpms_Java11', 'Run Jpms
       steps {
         gradle {
           rootBuildScriptDir(commonJobProperties.checkoutDir)
-          tasks(':sdks:java:testing:jpms-tests:integrationTest')
+          tasks(':sdks:java:testing:jpms-tests:postCommit')
```

Comment:
Noted below that coupling the various runners has some drawbacks. It is some boilerplate but can we factor it out into a few jobs: direct (candidate for precommit), flink (also candidate, but slower), dataflow (too slow for precommit).